packages/table/src/Table/TableWithVS.story.tsx

```diff
@@ -651,3 +539,91 @@ export const ExpandableContent: StoryFn<StoryTableProps> = args => {
     </>
   );
 };
+
+export const TallRows: StoryFn<StoryTableProps> = args => {
```

Comment:
In this story, you can continue to scroll even when there is no more rows. Is this expected? 

https://jmp.sh/mdpgthXk

In this video, I'm scrolling down and it looks like the scrollbar continues to move when there are no more rows.